app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx

```diff
@@ -130,6 +131,14 @@ export class SequenceAndBuildAndImages extends React.Component<IServerGroupHeade
           </a>
         )}
         {!!images && <ImageList {...this.props} />}
+        {!!ciBuild && ciBuild.jobUrl && (
+          <span>
+            &nbsp;({ciBuild.version})
```

Comment:
If you are just trying to ensure some whitespace is there, we've been doing `{' '}`

Reply:
okay, I'll fix that too.